Agents/GFAAgent/inputs/config/gfaagent.json

```diff
@@ -0,0 +1,52 @@
+{
+    "ServiceSpec": {
+        "Name": "gfaagent",
+        "TaskTemplate": {
+            "ContainerSpec": {
+                "Image": "ghcr.io/cambridge-cares/gfaagent:1.2.0",
+                "Env": [
+                  "STACK_NAME=${STACK_NAME}",
+                  "DATABASE=postgres",
+                  "osmSchema=osm",
+                  "osmPoints=points",
+                  "osmPolygons=polygons"
+                ],
+                "Mounts": [
+                  {
+                    "Type": "bind",
+                    "Source": "../../../../../../Agents/GFAAgent/src/main/resources",
+                    "Target": "/resources"
+                  }
+                ],
```

Comment:
As far as I understand, this is redundant, bind mounts are used for data files that should not be compiled with the code. In this case, all the files needed are already compiled together with the code.

For the obda file, you can read it like this:
```
Path obdaFile= new ClassPathResource("cost.obda").getFile().toPath();
OntopClient ontopClient = OntopClient.getInstance();
ontopClient.updateOBDA(obdaFile);
```
where ClassPathResource is org.springframework.core.io.ClassPathResource

You can get the input stream of the csv file easily via
`new ClassPathResource("cost_ontobuiltenv.csv").getInputStream();`

Reply:
sometimes I found the code can't read files or can't find class path, I am not sure the reason so I keep "Mounts" in config json. If you suggest to remove it, I am ok to do it.

Reply:
Never mind, keep this